docs/user-guide/creating-application/deployment-template/rollout-deployment.md

```diff
@@ -465,15 +465,25 @@ This is used to give arguments to command.
 command:
   enabled: false
   value: []
+  workingDir: {}
 ```
 
-It contains the commands for the server.
+It contains the commands to run inside the container.
 
 | Key | Description |
 | :--- | :--- |
 | `enabled` | To enable or disable the command. |
 | `value` | It contains the commands. |
+| `workingDir` | It is used to specify the working directory where commands will be executed. |
 
+**Note:** workingDir parameter will only work if the following conditions are satisfied.
```

Comment:
Too complicated, just mention that it only works if command is enabled or even that is self understood as it is a part of command section.

Reply:
Okay, making changes on it.